remote-api/src/main/java/org/alfresco/rest/api/impl/TypesImpl.java

```diff
@@ -151,9 +148,8 @@ public Type convertToType(TypeDefinition typeDefinition, List<String> includes)
         }
         catch (Exception e)
         {
-            logger.warn("Failed to parse Type" + typeDefinition.getName());
+            throw new AlfrescoRuntimeException("Failed to parse Type" + typeDefinition.getName());
```

Comment:
Thanks for changing this but looks like you've lost the space in the error message.